docs/dapp/opl/router-protocol/pingpong/README.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
The pingpong example chapters got pretty deep in the menu. I propose we flatten the last level to sth like:

OPL -> Router protocol -> README, Deploy, Approve, Communicate

Now the deploy, approve and communicate chapters are generic for any router app out there, right? Therfore, I would only mention in the beginning of the deploy chapter that we're going to use a "PingPong"-like example. I wouldn't build the whole guide around it, but write more general on deploy, approve and communicate actions. Ofc you can still use all the existing material (screenshots, code snippets).